crates/dataflow-jit/src/main.rs

```diff
@@ -104,6 +104,8 @@ fn run(program: &Path, config: &Path) -> ExitCode {
 
     let (mut demands, mut inputs) = (Demands::new(), Vec::with_capacity(config.inputs.len()));
     for (name, input) in config.inputs {
+        // Allow JIT file to specify unused inputs
+        if !source_names.contains_key(&name) { continue; }
         let (node, layout) = source_names[&name];
```

Comment:
```suggestion
        let (node, layout) = if let Some((node, layout)) = source_names.get(&name) {
            (node, layout)

        // Allow specifying unused inputs
        } else {
            continue;
        };
```

Reply:
can you incorporate this change in some of your newer PRs?